core/shared/src/main/scala/org/http4s/headers/Access-Control-Allow-Headers.scala

```diff
@@ -39,8 +41,8 @@ object `Access-Control-Allow-Headers` {
       parse
     )
 
-  implicit val headerSemigroupInstance: cats.Semigroup[`Access-Control-Allow-Headers`] =
-    (a, b) => `Access-Control-Allow-Headers`(a.values.concatNel(b.values))
+  implicit val headerSemigroupInstance: cats.Monoid[`Access-Control-Allow-Headers`] =
```

Comment:
We could go ahead and rename this for accuracy, since we're breaking compatibility.

Reply:
Yeah, I totally missed that